security/advancedtls/advancedtls.go

```diff
@@ -33,18 +33,21 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
-// VerificationFuncParams contains the parameters available to users when implementing CustomVerificationFunc.
+// VerificationFuncParams contains the parameters available to users when implementing |CustomVerificationFunc|.
+// The fields contain in this struct are used when making custom verification decisions, and are hence read-only.
+// Setting these fields wouldn't have any effect.
 type VerificationFuncParams struct {
+	// This field is only meaningful for client side. On server side, this field would be an empty string.
 	ServerName     string
 	RawCerts       [][]byte
 	VerifiedChains [][]*x509.Certificate
 }
 
-// VerificationResults contains the information about results of CustomVerificationFunc.
+// VerificationResults contains the information about results of |CustomVerificationFunc|.
```

Comment:
Please make sure that |.| is used everywhere for consistency. I'm fine removing these bars, `|`.

Reply:
Also, please make sure that gRPC-Go uses this style in the comments. We should try to be consistent with that gRPC-Go does.

Reply:
SG.  I didn't find any bars used in the gRPC-go implementation, so I will just remove all of them.